modules/vpc/output.tf

```diff
@@ -0,0 +1,11 @@
+output "infrastructure_subnets" {
+  value = {
+    for k, s in aws_subnet.aap_infrastructure_subnets : k => s.id
+  }
+
+  depends_on = [aws_subnet.aap_infrastructure_subnets]
+}
+
+output "infrastructure_igw" {
+  value = aws_internet_gateway.aap_infrastructure_igw.id
+}
```

Comment:
```suggestion
}

output "deployment_id" {
  value = var.deployment_id == "" ? random_string.deployment_id[0].id : var.deployment_id
}

```
tflint wanted this output here instead of the root output.tf. Guessing because it's actually used / set in this module.

Reply:
Sorry, I updated the suggestion to fix that. There's another way though. Keep the deployment_id in the root outputs.tf and add a description makes this happy. That's after the file name changes to include the missing `s`.